.travis/build.sh

```diff
@@ -18,14 +18,16 @@ exitIfError() {
   [ "$EXIT" != "0" ] && exit $EXIT
 }
 
+arch=$(uname -m)
+
 # The first segment of the version number is '1' for releases < 9; then '9', '10', '11', ...
 JAVA_MAJOR_VERSION=$(java -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
 if [ ${JAVA_MAJOR_VERSION} -gt 1 ] ; then
   export JAVA_VERSION=${JAVA_MAJOR_VERSION}
 fi
 
-if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
-  # some parts of the workflow should be done only once on the main build which is currently Java 8
+if [ "$arch" == 'x86_64' ] || [ "$arch" == 'arm64' ] ; then
+  # some parts of the workflow should be done only once on the main build which is currently Java 11
```

Comment:
Should you check the Java version here?

Reply:
Actually, the question is do we want to run as many of the tests on as many of the platforms as possible.

I'm not sure but IIRC the Arquillian Cube based testsuite only worked on Java 8, that has been fixed by moving to TestContainers.
Another thing I remember is that the testsuite run would take so long or produce so much output that it triggered some limits and was killed by Travis.
I guess we can try again running the full testsuite on both Java 11 and Java 17, and if that will hit limits then I suggest we limit the testsuite to Java 11, that's due to the next paragraph ...

We have to decide which class version libraries we want to push to maven upon release.
I would still prefer Java 8 as that makes the jars most widely usable, but we decided to move on from there. Therefore I suggest Java 11 for the same reason.

Reply:
I don't think anyone is prescribing what Java versions should Oauth support. We need Java 11 as a minimum. But if you want to support Java 8, then fine. But Java 8 is super old, Kafka will stop supporting it soon etc. So I think it is time to cut it. But even if you want to release with Java 8 as a target version, we need to have testing with Java 11 and Java 17

Reply:
Ok, I added `openjdk8` back to the Travis build, and reworked logic so that testsuite is run for all java versions on all platforms. Only Java 8 build on `x86_64` will be published if it's a release build.